src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,75 @@
+
+`init.proxy.beacon` is used by both installation methods to allow proxying of observability data through a different domain. For example, if this field is set to `nrbeacon.abc.com`, all observability data will be sent to `https://nrbeacon.abc.com/<observability endpoint>` instead of what is configured automatically on the New Relic account and found in the `NREUM.info.beacon` configuration.
+
+<Callout variant="important">
+These configurations should follow a pattern of `<domain>[:<port>][/<path>]`. The protocol `https://` and ending slash **should be omitted**. The `port` and `path` are optional and will be respected.
+</Callout>
+
+
+<Callout variant="tip">
+The browser agent team recommends using different values for each proxy configuration. This may mean each configuration is a different subdomain or possibly each contains a different path. This will make configuring the proxy server a lot easier.
+</Callout>
+
+## About your proxy server [#expectations]
+
+The New Relic browser agent team provides neither a cloud proxy service nor support on such service application. **You must make and maintain your own proxy servers.**
+
+To that effect, some assumptions are made about your proxy that must hold true for full functionality:
+
+1. As mentioned above, if a proxy is used to load the agent code, it should fetch them from our CDN and pass it back to the client.
+  a. Our current CDN is `https://js-agent.newrelic.com/`. E.g., for v1.240.0, requests to `https://yourproxy.com/nr-spa.3b61d78f-1.240.0.min.js` should yield the same response as from the upstream `https://js-agent.newrelic.com/nr-spa.3b61d78f-1.240.0.min.js`.
+2. The beacon proxy forwards client payloads to New Relic.
+  a. Your original endpoint can be found in your browser application settings page -- look for `;NREUM.info={beacon:"_____",...` in the copy snippet. For most apps, this would be `(https://)bam.nr-data.net`. However, it could be different for some customers for compliance and legal reasons.
+  b. E.g., requests to `https://yourproxy.com/1/asdf?a=123` should be passed through to `https://bam.nr-data.net/1/asdf?a=123`.
+3. Requests and responses through the proxy server should not be modified.
+4. The client IP is passed in the `X-Forwarded-For` HTTP header from the proxy to New Relic servers. This is needed for the geolocation features in New Relic to work correctly.
+  a. You can exclude this if user privacy is of concern. In the case that this field is missing, our geolocation will simply view your proxy's location as the "client".
+5. Your system has the capability and capacity to handle the amount of bandwidth and traffic generated by observability data collection.
+  a. This is entirely dependent upon the usage of your site.
```

Comment:
```suggestion
* The beacon proxy forwards client payloads to New Relic.
  * Your original endpoint can be found in your browser application settings page -- look for `;NREUM.info={beacon:"_____",...` in the JavaScript snippet. For most apps, this would be `(https://)bam.nr-data.net`. However, it could be different for some customers for compliance and legal reasons.
  * For example, requests to `https://yourproxy.com/1/asdf?a=123` should be passed through to `https://bam.nr-data.net/1/asdf?a=123`.
* Requests and responses through the proxy server should not be modified.
* The client IP is passed in the `X-Forwarded-For` HTTP header from the proxy to New Relic servers. This is needed for the geolocation features in New Relic to work correctly.
  * You can exclude this header if you're concerned with user privacy. If this field is missing, our geolocation will simply view your proxy's location as the "client".
* Your system should be able to handle the bandwidth and traffic generated by observability data collection, but the amount of data generated will depend on your site's usage.
```